tests/option/eckit_test_option_cmdargs.cc

```diff
@@ -1015,6 +1015,9 @@ CASE("test_eckit_option_cmdargs_value_with_equals") {
 #ifdef __clang__
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#elif defined(__INTEL_COMPILER)
+#pragma warning ( push )
+#pragma warning ( disable:1786 )
```

Comment:
Short description on this warning

Reply:
The description is on line 1014 just above. It is hidden by this diff.